Exiled.API/Features/Map.cs

```diff
@@ -397,6 +397,42 @@ public static void UnlockAllDoors()
                 door.ChangeLock(DoorLockType.None);
         }
 
+        /// <summary>
+        /// Gets an random <see cref="Room"/>.
+        /// </summary>
+        /// <returns><see cref="Room"/> object.</returns>
+        public static Room GetRandomRoom() => Rooms[Random.Range(0, Rooms.Count)];
+
+        /// <summary>
+        /// Gets an random <see cref="Camera079"/>.
+        /// </summary>
+        /// <returns><see cref="Camera079"/> object.</returns>
+        public static Camera079 GetRandomCamera() => Cameras[Random.Range(0, Cameras.Count)];
+
+        /// <summary>
+        /// Gets an random <see cref="Door"/>.
+        /// </summary>
+        /// <returns><see cref="Door"/> object.</returns>
+        public static Door GetRandomDoor() => Doors[Random.Range(0, Doors.Count)];
```

Comment:
Recommend:
`GetRandomDoor(bool onlyUnbroken = false)` to filter out broken doors.